CODE_OF_CONDUCT.md

```diff
@@ -39,7 +39,7 @@ This Code of Conduct applies within all community spaces, and also applies when
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the community leaders responsible for enforcement at [conduct@lilypond.org](mailto:conduct@lilypond.org). All complaints will be reviewed and investigated promptly and fairly.
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the community leaders responsible for enforcement at [lilypond.conduct@gmail.com](mailto:lilypond.conduct@gmail.com). All complaints will be reviewed and investigated promptly and fairly.
```

Comment:
i'm not quite sure that this email address works, I am not sure that we run an SMTP server. Perhaps it's better to explicitly spell out who the people responsible for CoC are.

Reply:
+1 for simplicity. I've found that git does this as well - that is, listing members individually (https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=8a104f8b5867c682d994ffa7a74093c54469c11f).